lib/rename-version.js

```diff
@@ -123,7 +124,7 @@ if (fs.existsSync(currentSidebarFile)) {
   fs.renameSync(currentSidebarFile, newSidebarFile);
   let sidebarContent = fs.readFileSync(newSidebarFile, 'utf8');
   sidebarContent = sidebarContent.replace(
-    new RegExp(`version-${RegExp.escape(currentVersion)}-`, 'g'),
+    new RegExp(`version-${escapeRegexp(currentVersion)}-`, 'g'),
```

Comment:
I think we can escape it using a library instead of our own escape function. Its alrd in our dependencies

Reply:
I'll look into that. Could you point me to which library it is?